### DISCUSSION TITLE
Which Verisure alarm system does this plugin support?

### DISCUSSION THREAD

**Discussion by peril49** [ptz0n/homebridge-verisure#132] [Q&A] [answered]
Does this support the newer Verisure alarm system?
![image](https://user-images.githubusercontent.com/4653210/146288874-e27fd5af-71b2-4c73-9429-44d5e39c6191.png)

**Comment by ptz0n** [comment, marked as answer]
I'm not sure which one you consider to be the newer one of these two.

This plugin have support for the system to the right (Vbox).
